### ISSUE TITLE
review: add a placeholder command at x/review/git-review?

### ISSUE THREAD

**Issue by minux** [golang/go#9499]
Given that we can't change https://golang.org/doc/contribute.html#Code_review
right now, people will try to install x/review/git-review. Does it make sense
to provide a dummy command at x/review/git-review to remind the user to
use x/review/git-codereview instead?

**Comment by bradfitz** [comment]
Why can't we change it now? @adg could.

**Comment by minux** [comment]
Yes, we can change the website, but what about user reading doc/contribute.html
in go1.4 binary releases?

**Comment by bradfitz** [comment]
I suppose, but I doubt many people do that.

**Comment by minux** [comment]
we don't need to actually provide a dummy command, we can put two Go files
at x/review/git-review with conflicting package names, so that go get fails with
some meaningful message. And we should update the website if possible.

// moved.go
package command_has_renamed // go get golang.org/x/review/git-codereview instead

// moved2.go
package moved

something like this, so that we don't actually install a git-review binary.

**Comment by minux** [comment]
Even better, if we could make an exception to include non-essential
fixes to minor releases and cherrypick CL 1742, 1402, 1743 to
release-branch.go1.4 when we release 1.4.1. (Possibly also CL 2087.)

I will label this issue 1.4.1 to remind us.

**Comment by adg** [comment]
I don't think it really matters. I fixed the instructions on the web site, and fixed the deploy script so we won't get the old instructions again.
